src/mutation/__tests__/mutation.js

```diff
@@ -65,6 +65,7 @@ var simpleRootValueMutation = mutationWithClientMutationId({
       type: GraphQLInt
     }
   },
+  // $FlowFixMe: check if Relay narrowed GraphQLResolveInfo.rootValue to Object
```

Comment:
Test does not affect src code ,just suppress it.
It is valid when using a flowType-outputed version of GraphQL .